learner_pathway_progress/utilities.py

```diff
@@ -77,11 +121,14 @@ def update_learner_pathway_progress(user_id, course_id):
         for learner_pathway_uuid in pathways_linked_with_course:
             pathway_snapshot = get_pathway_snapshot(user, learner_pathway_uuid)
             if pathway_snapshot:
-                LearnerPathwayMembership.objects.get_or_create(
-                    user=user,
-                    learner_pathway_uuid=learner_pathway_uuid
-                )
-                pathway_progress, _ = LearnerPathwayProgress.objects.get_or_create(
+                enterprise_learner_linked_with_course_run = get_learner_enterprises_for_course_run(user_id, course_key)
```

Comment:
`get_learner_enterprises_for_course_run` function works oky with course key instead of course_run key?